tensorflow/python/kernel_tests/slice_op_test.py

```diff
@@ -146,6 +147,27 @@ def testSingleDimension(self):
         slice_val = self.evaluate(slice_t)
         self.assertAllEqual(slice_val, inp[lo:hi])
 
+  def test3Dimension(self):
+    with self.session() as sess:
+      input_shape = [8, 16, 16, 16, 8]
+      inputs = random_ops.random_normal(input_shape,
+                                        dtype=dtypes.float32,
+                                        seed=0)
+      filter_shape = [1, 1, 1, 8, 8]
+      filters = random_ops.random_normal(filter_shape,
+                                         dtype=dtypes.float32,
+                                         seed=0)
+
+      conv_t = nn_ops.conv3d(inputs,
+                             filter=filters,
+                             strides=[1, 1, 1, 1, 1],
+                             padding="VALID")
+      slice_t = array_ops.slice(conv_t, [0, 1, 1, 1, 0], [1, 1, 1, 1, 8])
+      result = slice_t.eval()
```

Comment:
`slice_t.eval()` doesn't work in eager mode. Please use `self.evaluate` instead.
```suggestion
      result = self.evaluate(slice_t.eval)
```